src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -185,7 +185,7 @@ export const CollectionNftsLoading = () => (
 export const CollectionNftsAndMenuLoading = () => (
   <InfiniteScrollWrapper>
     <Column alignItems="flex-start" position="relative" width="full">
-      <Row marginY="12" gap="12">
+      <Row marginY="12" gap="12" marginBottom="40">
```

Comment:
the space between the nfts and the filter button/ search and sort is a bit off for the loading state.  This makes the transition look more natural